app/models/member.rb

```diff
@@ -0,0 +1,4 @@
+class Member < ApplicationRecord
+	validates :email, :uniqueness => { :case_sensitive => true }
```

Comment:
I think this needs to be `{ :case_sensitive => false }`? `true` is the default, less strict behavior.

Reply:
Oops, I misread [your comment].(https://github.com/firstdraft/draft_generators/issues/65#issuecomment-527585045). You definitely wrote case-**in**sensitive.